Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -3537,7 +3537,7 @@ enum class WebCore::ResourceLoadPriority : uint8_t {
     Opaqueredirect
 };
 
-[Nested] enum class WebCore::ResourceResponseBaseSource : uint8_t {
+[CustomHeader] enum class WebCore::ResourceResponseBaseSource : uint8_t {
```

Comment:
If we put this in a struct, it can remain Nested.

Reply:
Will do that asa follow on